aten/src/ATen/native/quantized/cudnn/Conv.cpp

```diff
@@ -294,18 +373,35 @@ void raw_cudnn_convolution_forward_out(
 //
 // output Tensor will be a clampped int8 Tensor
 // both act and weight will be int8 Tensor
-//
+/*
+Numerics:
+Operation:
+out_fp32 = conv_fp32(act_fp32, w_fp32, …)
```

Comment:
I think we also want to tie this closer to the code, we can write down the operations for each major op and their arguments, e.g. we have conv op, add bias op, requantize op etc.